### PR TITLE
Release/6.7.0

### PR DESCRIPTION
**Pull request by kopijunkie**
### Added

- Internal: Introduce `SdkOptionsProvider` component and `useSdkOptions()` hook for SDK options' single source of truth.
- Public: Added support for UI customizations in SDK configuration using `customUI` option. See README for details of supported customization options.
- Internal: Add Woopra tracking for UI customization option usage.

### Changed

- UI: Replaced internal button component with button from @onfido/castor-react.
- UI: Replaced some Sass variables with CSS variables to allow customization of colors and fonts.
- Public: Added new enterprise feature `logoCobrand`. When purchased and enabled allows integrator to provide their own logo image to be displayed alongside the Onfido logo.
- Internal: Use Node 14 LTS for Travis to be consistent with `.nvmrc` and `Dockerfile`.

### Fixed

- UI: Fix Camera Permission icon not displaying on iOS devices on Selfie/Liveness capture flow

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1384-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,9 +6,27 @@ This change log file is based on best practices from [Keep a Changelog](http://k
 This project adheres to [Semantic Versioning](http://semver.org/). Breaking changes result in a different MAJOR version. UI changes that might break customizations on top of the SDK will be treated as breaking changes too.
 This project adheres to the Node [default version scheme](https://docs.npmjs.com/misc/semver).
 
-## [next-version]
```
Comment (stefaniacardenas) [inline CHANGELOG.md:9]: We still want to have the `next-version` header at the top. Clicking on this header will open a link to the list future changes that are in development in comparison to `6.7.0`.

File: README.md [hunk 2]
```diff
@@ -806,6 +948,467 @@ VIDEO_FACIAL_CAPTURE_STEP_2 - User reached the 2nd challenge during "liveness vi
 UPLOAD - User's file is uploading
 ```
 
+## Premium Enterprise Features
```
Comment (stefaniacardenas) [inline README.md:951]: Out of scope: this readme is getting long, I wonder if it still makes sense to keep this info here or if we should create a separate .md file

File: CHANGELOG.md (hunk 1 above)
Comment (kopijunkie) [reply CHANGELOG.md:9]: Ah, ok. Looks like I misunderstood what you meant in our previous discussion about this. Will bring it back in.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1384-pr-onfido-sdk-ui-onfido.surge.sh

File: README.md (hunk 2 above)
Comment (kopijunkie) [reply README.md:951]: Agreed. I think we're eventually going to have all this on the Developer Hub & it should be broken up into smaller pages there, but in the meantime I can look into how we can break up the documentation here into separate Markdown files.

**Review by ethan605 — APPROVED** [review]

**Review by stefaniacardenas — APPROVED** [review]

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-1384-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://1384-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**